tests/lib/rules/define-props-declaration.js

```diff
@@ -10,8 +10,11 @@ const rule = require('../../../lib/rules/define-props-declaration')
 const tester = new RuleTester({
   languageOptions: {
     parser: require('vue-eslint-parser'),
-    ecmaVersion: 2020,
-    sourceType: 'module'
+    ecmaVersion: '2020',
```

Comment:
To fix the failing tests, this has to be reverted:

```suggestion
    ecmaVersion: 2020,
```